.gitignore

```diff
@@ -347,6 +347,7 @@ MigrationBackup/
 # Ionide (cross platform F# VS Code tools) working folder
 .ionide/
 packages/api/build/
```

Comment:
`failpoint-ts` is newly added by @1yefuwang1  and needs to add this line in *.gitignore* to avoid noisy build files being detected by git system.